cmp/grid/renderers/TagsRenderer.js

```diff
@@ -0,0 +1,12 @@
+import {isEmpty} from 'lodash';
+import {div, hbox} from '@xh/hoist/cmp/layout';
+import './TagsRenderer.scss';
+
+export function tagsRenderer(v) {
```

Comment:
Should we take this on step further and add a `tags` column partial in `cmp/grid/columns/Core.js`?  Does not seem any less general that the columns we define there.

Reply:
Makes sense, should we also add some defaulting for TAGS fields in GridModel for columns to use the renderer? Or is the JS default toString of a csv a better default?